requirements.txt

```diff
@@ -5,4 +5,5 @@ patsy>=0.4.0
 joblib>=0.9
 tqdm>=4.8.4
 six>=1.10.0
+h5py>=2.7.0
 enum34>=1.1.6; python_version < '3.4'
```

Comment:
I think this is because Enum34 is *only* needed for py34, but *not* needed for py35 and py36.